helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -51,6 +51,8 @@ generic-service:
       VISITS_CLIENT_SECRET: "VISITS_CLIENT_SECRET"
       INCENTIVES_CLIENT_ID: "INCENTIVES_CLIENT_ID"
       INCENTIVES_CLIENT_SECRET: "INCENTIVES_CLIENT_SECRET"
+      ACTIVITIES_CLIENT_ID: "ACTIVITIES_CLIENT_ID"
+      ACTIVITIES_CLIENT_SECRET: "ACTIVITIES_CLIENT_SECRET"
```

Comment:
Have these been created in live? Else this can't be merged since deployment will fail

Reply:
I raised https://dsdmoj.atlassian.net/browse/HAAR-1092 to cover it. Ill do dev myself if Simon doesnt get round to it tomorrow